src/sources/soundsourceffmpeg.cpp

```diff
@@ -16,7 +16,7 @@ constexpr AVSampleFormat kavSampleFormat = AV_SAMPLE_FMT_FLT;
 
 constexpr uint64_t kavChannelLayoutUndefined = 0;
 
-constexpr int64_t kavMinStartTime = 0;
+constexpr int64_t kavDefaultStreamStartTime = 0;
```

Comment:
What is the unit? Frame Indices?

Reply:
I am looking forward to having the FramePos class from #2861 to use in the rest of Mixxx.

Reply:
No, FFmpeg time units. That's why it is prefixed with "av".

Reply:
Until we have that we should add the unit it to the variable name. That makes it easier to spot issues during code review.

Reply:
It is named in the same way as the corresponding functions with the suffix "Time". The type also mirrors the FFmpeg type. I don't know of a better name.

Reply:
@uklotzde sorry I didn't see your comment when I replied. In that case it can stay like that.